net.granjow.slowmovideo.yaml

```diff
@@ -0,0 +1,75 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
```

Comment:
```suggestion
runtime-version: '5.15-22.08'
```

it's out now.

Reply:
unfortunately, it doesn't compile with the new compiler :(
https://github.com/flathub/flathub/pull/3280/commits/3f83be108e76b5c7297e89cb22c6e63cadcdec3a
Maybe https://github.com/slowmoVideo/slowmoVideo/commit/2a743063e38af6b4c747064243fc1f9b431d8c0e helps.
That's something to look at in the future.